src/main/webapp/WEB-INF/themes/desktop/sass/_typography.scss

```diff
@@ -379,9 +393,9 @@ example | extension
 //font decorations and JATS tags
 ///////////////
 
-.type{
-  font-weight:$bold;
-  &:after{content: ':'}
+.type {
+  font-weight: $bold;
+  &:after { content: ':'; }
```

Comment:
Since you're doing some reformatting of the stylesheets I wanted to note this one-liner.  Unsure if that goes against our style.